src/devices/Display/LedSegmentDisplay5641AS.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Iot.Device.Display;
+
+namespace Display
+{
+    /// <summary>
+    /// LED 4-digit 7-segment display, model 5641AS. This class handles driverless, direct-pin access over GPIO.
+    /// </summary>
+    /// <remarks>
+    /// Data Sheet:
+    /// http://www.xlitx.com/datasheet/5641AS.pdf
+    /// </remarks>
+    public class LedSegmentDisplay5641AS : ISevenSegmentDisplay, IDisposable
+    {
+        private readonly LedSegmentDisplay5641ASPinScheme _pinScheme;
+        private readonly GpioController _gpioController;
+        private readonly bool _shouldDispose;
+        private bool _disposedValue;
+
+        private List<GpioPin> _segments;
+        private List<GpioPin> _digits;
+
+        /// <inheritdoc />
+        public int NumberOfDigits => _pinScheme.DigitCount;
+
+        /// <inheritdoc />
+        public Segment this[int address] { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
```

Comment:
is there any way to recreate a Segment? So that it's make easy for the user and kind of easier?

Reply:
This has been updated in latest push - see  https://github.com/dotnet/iot/pull/2350/commits/e3f4cc05e0a007a5090d7955380df7bc77bbafa4